examples/MenuButtons.cc

```diff
@@ -0,0 +1,60 @@
+/*
+  Copyright (c) 2000 - 2012 Novell, Inc.
+
+  Permission is hereby granted, free of charge, to any person obtaining
+  a copy of this software and associated documentation files (the
+  "Software"), to deal in the Software without restriction, including
+  without limitation the rights to use, copy, modify, merge, publish,
+  distribute, sublicense, and/or sell
+  copies of the Software, and to permit persons to whom the Software is
+  furnished to do so, subject to the following conditions:
+
+  The above copyright notice and this permission notice shall be
+  included in all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT
+  SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+  DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+  OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR
+  THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+
+
+// Menu Button example demonstrating also shortcut resolver.
+//
+// Compile with:
+//
+//     g++ -I/usr/include/yui -lyui MenuButtons.cc -o MenuButtons
+
+#include "YUI.h"
+#include "YWidgetFactory.h"
+#include "YDialog.h"
+#include "YLayoutBox.h"
+#include "YEvent.h"
+#include "YMenuButton.h"
+#include "YMenuItem.h"
+
+int main( int argc, char **argv )
+{
+    YDialog    * dialog = YUI::widgetFactory()->createPopupDialog();
+    YLayoutBox * vbox   = YUI::widgetFactory()->createVBox( dialog );
+    YUI::widgetFactory()->createLabel( vbox, "Hello, World!" );
+    YMenuButton* top = YUI::widgetFactory()->createMenuButton( vbox, "Menu!" );
+    YMenuItem *inner_item = new YMenuItem("&menu1");
+    top->addItem(inner_item);
+    top->addItem(new YMenuItem("&menu2"));
```

Comment:
Please change this to &Menu2 so that we see whether the letter case resolution works.